unified-installer/astra-unified-installer.sh

```diff
@@ -753,6 +772,19 @@ fatal() {
     exit 1
 }
 
+# remove_kubectl_warnings_from_error_output will output kubectl warnings to stderr and actual errors
+# to stdout, allowing one to examine proper errors separately from warnings.
+# Usage: local -r actual_errors="$(echo "$my_error_output" | remove_kubectl_warnings_from_error_output)"
+remove_kubectl_warnings_from_error_output() {
```

Comment:
This is to fix a bug where some `kubectl apply` commands were reported as failed because kubectl warnings are output on the same channel as errors, making warnings appear as errors.